theme/brb/views/dataprivacy/privacy_cookies.hbs

```diff
@@ -0,0 +1,5 @@
+<h3>
```

Comment:
is this the same like from maintheme? same for the file below.

Reply:
No it's not. There's 1 sentence missing about cookies used for user behaviour tracking that we do in main.